src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
It would be better to name it `AreaCalculator`
and method `getArea()` or `calculateArea()`